packages/components/focus-trap/src/utils.ts

```diff
@@ -88,10 +88,6 @@ export const tryFocus = (
       isSelectable(element) &&
       shouldSelect
     ) {
-      if (element.tagName === 'INPUT') {
-        element.setSelectionRange(element.value.length, element.value.length)
```

Comment:
Because it was in the all-selected state before, it was impossible to judge the cursor position. When this version was optimized, this unit was processed. At that time, only the tag was judged as input, and the judgment of type was ignored.

Reply:
Hmmm... That does solve the problem, but I don't think we should manually control the cursor position in this case. Maybe we can save it from shifting focus by preventing the default event of `mousedown` of date table cell. I've updated a commit, WDYT?